app/lib/audit/catalog_to_moab.rb

```diff
@@ -45,10 +44,12 @@ def check_catalog_version
       compare_version_and_take_action
     end
 
-    alias storage_location storage_dir
-
     private
 
+    def storage_location
+      complete_moab.moab_storage_root.storage_location
```

Comment:
is there any advantage in memoizing this?

also - we're sure this won't cause multiple "which storage_root am i on?" queries (seems like it won't) ... cause those can be expensive.

Reply:
I... don't know! What do you think, @jmartin-sul?

Reply:
my recollection is that AR will essentially do some naive caching for us.  there'll be a query for the `#moab_storage_root` call, but it turns out we're already doing that in the constructor to get an `AuditResults` object:  https://github.com/sul-dlss/preservation_catalog/blob/5398420867e2c91f4dc594643dfc698500bf9fbd/app/lib/audit/catalog_to_moab.rb#L9-L13

and then i believe we'll be accessing the same instance of `MoabStorageRoot` until we call `#reload` on either `complete_moab` or `complete_moab.moab_storage_root`.

basically my recollection is that ActiveRecord doesn't re-query for accessing a field once the data has been retrieved and the object constructed.

Reply:
@ndushay What do you think? :point_up:

Reply:
@ndushay What say you re: :point_up: ?

Reply:
all good.  Go for it. Push the green button.